modules/new_installer.py

```diff
@@ -0,0 +1,82 @@
+def download(package, config):
+    # Imports
+    from configparser import ConfigParser
+    from configparser import ExtendedInterpolation
+    import modules.cross_platform as cp
+    from urllib import request
+    import subprocess
+    import shutil
+
+    # Config
+    os_platform = config['OS.platform']
+    remote_location = config['Remote.location']
+    remote_branch = config['Remote.branch']
+    file_extension = config['Remote.file_extension']
+
+    # Downloading
+    print('--> Downloading Package File')
+    file_name = package + file_extension
+    file_url = remote_location + os_platform + '/' + remote_branch + '/new-scripts/' + file_name
+    cp.chdir('scripts/', os_platform)
+    with request.urlopen(file_url) as response, open(
+        file_name, 'wb') as  package_file:
+            shutil.copyfileobj(response, package_file)
+
+    # Additional Resources
+    print('--> Downloading Additional Resources')
+    package_file = ConfigParser(interpolation=ExtendedInterpolation())
+    package_file.read(file_name)
+    package_download = package_file['Script']['download']
```

Comment:
The download section need a check because there are some packages that are not supposed to download anything, like [oh-my-zsh.sh](https://github.com/YetAnotherPackageInstaller/packages-linux-debian/blob/master/scripts/oh-my-zsh.sh).
It's not a problem to create a download field, we need to separate the download resource from the executable.
Instead of this:
```
sh -c "$(wget https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh -O -)"
```
this:
```
wget https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh -O install.sh
sh ./install.sh
```
or something like that.
We need to remember to do that although can be a problem.

Reply:
That's good to remember. We need to transfer the shell scripts to this new package type so that  we can run the new version.